docs/book.json

```diff
@@ -0,0 +1,40 @@
+{
+  "gitbook": "3.x.x",
```

Comment:
What's the version here used for? Should it be pinned?

Reply:
It's for specifying the gitbook version for plugins https://toolchain.gitbook.com/config.html. Not sure if it can be pinned. It does accept things like ">= 3.0.0"